core-blocks/audio/edit.js

```diff
@@ -102,6 +109,37 @@ class AudioEdit extends Component {
 					</Toolbar>
 				</BlockControls>
 				<InspectorControls>
+					<PanelBody title={ __( 'Preload' ) }>
+						<ButtonGroup aria-label={ __( 'Preload' ) }>
+							{ map( [
+								{
+									name: __( 'Auto' ),
+									key: 'auto',
+									attributeValue: 'auto',
+								},
+								{
+									name: __( 'Metadata' ),
+									key: 'metadata',
+									attributeValue: 'metadata',
+								},
+								{
+									name: __( 'None' ),
+									key: 'none',
+									attributeValue: undefined,
```

Comment:
```
attributeValue: undefined,
```
Worth noting I was stuck on this for a little bit. Originally, I tried `attributeValue: false` but ended up with `preload="false"` in the saved `<audio>` attribute.

Reply:
If `null` works I might think that is ever-so-slightly more semantically correct? If it worked I think it might be nicer, but yeah this is fine 😄

Reply:
`null` didn't work in my original testing.